<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationExpirationReminder</class>
 <widget class="QDialog" name="RegistrationExpirationReminder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lab_message">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_verify">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_verify</sender>
   <signal>clicked()</signal>
   <receiver>RegistrationExpirationReminder</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
